lib/screen/result_screen.dart

```diff
@@ -54,64 +54,67 @@ class _ResultScreenState extends State<ResultScreen> with SingleTickerProviderSt
     return Scaffold(
       body: Column(
         children: [
+          Padding(
+            padding: const EdgeInsets.only(top: 80.0),
+            child: FadeTransition(
+              opacity: _fadeAnimation,
+              child: Align(
+                alignment: Alignment.topCenter,
+                child: Text(
+                  'Hey, ${widget.userName}!',
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.teal.shade800,
+                  ),
+                ),
+              ),
+            ),
+          ),
+
           Expanded(
-            child: Center(
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  SizedBox(height: 30),
-                  FadeTransition(
-                    opacity: _fadeAnimation,
-                    child: Text(
-                      'Hey, ${widget.userName}!',
-                      style: TextStyle(
-                        fontSize: 24,
-                        fontWeight: FontWeight.bold,
-                        color: Colors.teal.shade800,
+            child: Column(
+              children: [
+                Spacer(flex: 1), // Pushes the circle to the middle
```

Comment:
If you are using this just to display the circle in the centre, you can make use of the Column's main axis alignment properties